website/content/docs/secrets/pki/troubleshooting-acme.mdx

```diff
@@ -0,0 +1,198 @@
+    --eab-hmac-key MHcCAQE... additional data elided ...
+```
+
+Ensure that the ACME directory passed to the ACME client matches that
+fetched from the Vault.
+
+## Debugging EAB directory bindings
+
+If an EAB account token is incorrectly used with the wrong directory, the
+ACME server will reject the request with an error about insufficient
+permissions.
+
+### Symptoms
+
+When initializing a new account against this Vault server, an error might
+appear like:
+
+> The client lacks sufficient authorization :: failed to verify eab
+
+This is caused by requesting an EAB from a directory not matching the
+one the client used.
+
+### Resolution
+
+Ensure the requested EAB token matches the directory. For a given directory
+at `/some/path/acme/directory`, fetch EAB tokens from
+`/some/path/amce/new-eab`. The remaining resolution steps are the same as
+for [debugging account registration
+failures](#debugging-account-registration-failures).
+
+## Debugging issuance failures
+
+When Vault can't verify the server's identity through the client's requested
+[challenge type](/vault/api-docs/secret/pki#acme-challenge-types) (`dns-01`,
+`http-01`, or `tls-alpn-01`), Vault will not issue the certificate requested
+by the client.
+
+### Symptoms
+
+In the Vault logs or returned by the client, an error might occur like:
+
+> ACME validation failed for a465a798-4400-6c17-6735-e1b38c23de38-tls-alpn-01: ...
+
+this indicates that the server was unable to validate this challenge accepted
+by the client.
+
+### Resolution
+
+Ensure that DNS is configured correctly from the Vault server's perspective,
+including setting [any custom DNS resolver](/vault/api-docs/secret/pki#dns_resolver).
+
+Ensure that any firewalls are set up to allow Vault to talk to the relevant
+systems (the DNS server in the case of `dns-01`, port 80 on the target
+machine for `http-01`, or port 443 on the target machine for `tls-alpn-01`
+challenges).
+
+## Debugging account revocation
+
+Vault, via the [tidy mechanics](/vault/api-docs/secret/pki#tidy_acme), will
+periodically remove stale accounts.
+
+### Symptoms
+
+When Vault has removed an account that it has deemed stale, if a client
+attempts to reuse this account, an error will occur such as:
+
+> The client lacks sufficient authorization: account in status: revoked
+
+### Resolution
+
+Refer to the ACME client's documentation for removing cached local
+configuration and setup a new account, specifying any EABs as required.
+
+## Tutorial
+
+Refer to the [Build Your Own Certificate Authority (CA)](/vault/tutorials/secrets-management/pki-engine)
+guide for a step-by-step tutorial.
+
+Have a look at the [PKI Secrets Engine with Managed Keys](/vault/tutorials/enterprise/managed-key-pki)
+for more about how to use externally managed keys with PKI.
+
+## API
+
+The PKI secrets engine has a full HTTP API. Please see the
+[PKI secrets engine API](/vault/api-docs/secret/pki) for more
+details.
```

Comment:
Same questions as above:
Why is a note about the API relevant to a troubleshooting doc? What problem are you trying to solve by including this section?

Reply:
This is a common footer included in all docs pages just about.